README.md

```diff
@@ -1,68 +1,58 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Coding to Learn and Create - Inclusive Coding Environment
 
-## Available Scripts
+The Inclusive Coding Environment is a new educational coding tool designed for learners with disabilities. Learners can create code sequences that control the movement of a Bluetooth-connected robot. The Environment can be used with a mouse, touchscreen, or keyboard, and supports assistive technologies such as screen readers. It is a web application that works in the Chrome browser.<br>
 
-In the project directory, you can run:
+![inclusive coding environment UI](./inclusive-coding-env.png)
 
-### `npm start`
+[Live Environment](https://c2lc-dev.web.app/)
```

Comment:
To try the latest version of the Inclusive Coding Environment, visit our [development build](https://c2lc-dev.web.app/). Or something like that?

Reply:
Should we say 'latest development version'? latest version might refer to the latest release

Reply:
Sure, that sounds great.